models/inherit.py

```diff
@@ -28,20 +30,45 @@ def current_user_rents(self):
         
     @api.multi
     def get_rent_count(self):
-        data_obj = self.env['bike.rent']
-        print("*****************************************")
-        print(data_obj.partner_id.name)
-        print("*****************************************")
-
         for record in self:
+            record._compute_rent_count = len(record.partner_ids)
+            print(record._compute_rent_count)
+
+class account_invoice(models.Model):
+    _inherit = 'account.invoice'
+    return_date = fields.Date(string='Bike return date')
+    
+    @api.multi
+    def return_date(self):
+        bike_rent_obj = self.env['sale.order']
+        self.return_date = bike_rent_obj.rent_stop
```

Comment:
1. Method called the same as variable, this should cause all sorts of errors.
2. Why are you even inside invoices?
3. You are taking an empty `sale.order` object, taking its `rent_stop` value (which is `False`) and assigning it.
4. Method declared as capable to handle a recordset, but inside is a singleton assignment.